ci/update_securityhub_suppressions/suppressions.yml

```diff
@@ -302,6 +304,8 @@ Suppressions:
         - arn:aws:lambda:eu-west-2:776473272850:function:dev-get-event
         - arn:aws:lambda:eu-west-2:776473272850:function:dev-get-events
         - arn:aws:lambda:eu-west-2:776473272850:function:dev-delete-event
+        - arn:aws:lambda:eu-west-2:776473272850:function:dev-gro-ingestion-lambda-function-publish-event
+        - arn:aws:lambda:eu-west-2:776473272850:function:demo-gro-ingestion-lambda-function-publish-event
```

Comment:
I don't think this note applies to these functions?

Reply:
I've removed test, but it does. The lambdas are proxies that only access our system over the public internet